kuma/urls.py

```diff
@@ -19,23 +16,21 @@
 admin.autodiscover()
 
 urlpatterns = [re_path("", include("kuma.health.urls"))]
-# The non-locale-based landing URL's
-urlpatterns += [re_path("", include("kuma.landing.urls"))]
-urlpatterns += i18n_patterns(
-    re_path(
-        r"^events",
-        # Here the "shared_cache_control" decorator is an optimization. It
-        # informs the CDN to cache the redirect for a month, so once this URL
-        # has been requested by a client, all other client requests will be
-        # redirected by the CDN instead of this Django service.
-        shared_cache_control(s_maxage=MONTH)(
-            RedirectView.as_view(
-                url="https://community.mozilla.org/events/", permanent=False
-            )
-        ),
-        name="events",
-    ),
-)
```

Comment:
I'm thinking we should keep this redirect. I don't feel strongly about it though.